frontend/src/prompted_readings/analysis_view.js

```diff
@@ -28,14 +28,32 @@ class AnalysisView extends React.Component {
     render() {
         if (this.state.analysis !== null) {
             const {  // object destructuring:
-                total_view_time,
+                total_view_time, all_responses
             } = this.state.analysis;
 
             return (
-                <div>
+                <div style = {{padding: '1em'}}>
```

Comment:
We're going to use Bootstrap classes for styling, rather than hardcoding CSS like this. I think @amy-lei was starting work on styling the container for the whole page, so let's cut this here for now.